examples/visualization/plot_ssp_projs_sensitivity_map.py

```diff
@@ -23,7 +24,9 @@
 fname = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
 ecg_fname = data_path + '/MEG/sample/sample_audvis_ecg-proj.fif'
 
-fwd = read_forward_solution(fname, surf_ori=True)
+fwd = read_forward_solution(fname)
+fwd = convert_forward_solution(fwd, surf_ori=True)
```

Comment:
No, the warning will only pop up if `force_fixed=True` and `use_cps=None`. As the default is `force_fixed=False`, we only need to give `use_cps` explicitely, while generating fixed forwards. I can add that.